### PR TITLE
type def fixes

### PR DESCRIPTION
**Pull request by natebrennand**
Sorry, I didn't quite do my due diligence before opening #62. Found 3 more small things.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -296,7 +296,7 @@ declare module "ldclient-node" {
      * @param callback
      *  Will be called after the upsert operation is complete.
      */
-    upsert: (key: string, flag: LDFlag, callback?: () => void) => void;
+    upsert: (key: string, flag: LDFlagValue, callback?: () => void) => void;
```
Comment (natebrennand) [inline index.d.ts]: There was no definition for `LDFlag`, but there was for `LDFlagValue`.

File: index.d.ts [hunk 2]
```diff
@@ -349,7 +349,7 @@ declare module "ldclient-node" {
      */
     variation: (key: string, user: LDUser, defaultValue: LDFlagValue, callback?: (err: any, res: LDFlagValue) => void) => void;
 
-    toggle: typeof variation;
+    toggle: (key: string, user: LDUser, defaultValue: LDFlagValue, callback?: (err: any, res: LDFlagValue) => void) => void;
```
Comment (natebrennand) [inline index.d.ts]: I don't know of a terse way to replicate a function type definition. If `toggle` is deprecated, maybe the solution is to just remove it from the type definitions altogether?

Comment (dlau) [inline index.d.ts]: I think we should keep it for legacy compatibility until we remove it.

**Comment by dlau** [comment]
Thanks for this. 👍

**Comment by dlau** [comment]
published to npm `ldclient-node@3.0.13`

**Pull request merged**